JamfSync/Model/Jcds2Dp.swift

```diff
@@ -200,33 +207,453 @@ class Jcds2Dp: DistributionPoint {
         } else {
             fileUrl = file.fileUrl
         }
+        
         guard let fileUrl else { throw DistributionPointError.badFileUrl }
+        
+        
+        _ = disableSleep(reason: "start upload")
+
+        if let fileProperties = try? FileManager.default.attributesOfItem(atPath: fileUrl.path(percentEncoded: false)) {
+            if let size = fileProperties[FileAttributeKey.size] as? NSNumber {
+                let uploadFileSize = size.doubleValue
+                Chunk.all = Int(truncating: size)
+                Chunk.numberOf = Int(uploadFileSize)/Chunk.size
+                if Chunk.all % Chunk.size > 0 {
+                    Chunk.numberOf += 1
+                }
+                LogManager.shared.logMessage(message: "File will be split into \(Chunk.numberOf) parts.", level: .debug)
+                if uploadFileSize > 32212255000 {
```

Comment:
Best to put this magic number in a variable at the top of the class with a descriptive name (like maxUploadSize), so this code is self documenting and easier to change later.